src/Components/Header.css

```diff
@@ -0,0 +1,44 @@
+.navbarContainer{
+    margin: 0 1rem;
+    background: #ffff;
```

Comment:
hex colors should either have 6 characters after the # or 3.  I don't think 4 characters is valid